pkg/archive/targz/targz.go

```diff
@@ -25,7 +25,7 @@ func (a Archive) Close() error {
 
 // New tar.gz archive
 func New(target io.Writer) Archive {
-	gw := gzip.NewWriter(target)
+	gw, _ := gzip.NewWriterLevel(target, gzip.BestCompression)
```

Comment:
should we add a comment saying the error will be nil since the compression level is valid on both usages of this? 

just so its clear why the error is being ignored 🤔

Reply:
I think add a comment is a good idea.  Do I need to submit another PR with a comment?

Reply:
I added a comment about this.